misc/git-hooks/pre-commit

```diff
@@ -75,8 +75,9 @@ function check_check_forbidden_dependencies() {
         return 1
       fi
     done
+    # FIXME: (@leonardo) temporally allow the `openssl`, it's currently being brought by tor-proto -...-> native-tls.
```

Comment:
:fearful:

Reply:
Oh, BTW. This file is auto-generated...

The right way to change stuff here is to tweak flakebox config like I do in: https://github.com/fedimint/fedimint/pull/5473/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R83

Reply:
Actually, the original file in nix/check-forbidden-deps.sh

Reply:
Great, I'll update that then.
I now noticed that the comment is outdated, it's due to a problem on the latest jsonrpsee release, which might be fixed by this issue: https://github.com/paritytech/jsonrpsee/issues/1407

Reply:
if add openssl please gate it behind a cargo feature flag, mobile apps build break otherwise

Reply:
It'll be fixed by https://github.com/fedimint/fedimint/pull/5544, depends on new jsonrpsee release with https://github.com/paritytech/jsonrpsee/pull/1419

Reply:
I fixed it upstream, and now with the latest jsonrpsee release, this is not a problem anymore.